src/rules/at-use-no-redundant-alias/index.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const { utils } = require("stylelint");
+const namespace = require("../../utils/namespace");
+const ruleUrl = require("../../utils/ruleUrl");
+
+const ruleName = namespace("at-use-no-redundant-alias");
+
+const messages = utils.ruleMessages(ruleName, {
+  rejected: "Unnecessary explicit namespace"
```

Comment:
Check the developer guide for the format:
https://stylelint.io/developer-guide/rules#add-problem-messages